CHANGELOG.md

```diff
@@ -12,6 +11,11 @@ All notable changes to this project will be documented in this file.
 - Remove references to backup/groups [#1299](https://github.com/wazuh/wazuh-packages/pull/1299)
 - Add Azure's orm file to Solaris 11 SPEC [#1453](https://github.com/wazuh/wazuh-packages/pull/1453)
 
+## [v4.3.1]
```

Comment:
Add the rest of the versions

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/1677/commits/a50ea9847b32051d1fa02e721a6cfe2ffd557864